analysis/create_project_actions.R

```diff
@@ -43,9 +43,21 @@ describe <- FALSE # This prints descriptive files for each dataset in the pipeli
 # List of models excluded from model output generation
 
 excluded_models <- c(
-  "cohort_vax-sub_age_18_39-dem_lb",
-  "cohort_unvax-sub_age_18_39-dem_lb",
-  "cohort_prevax-sub_age_18_39-dem_lb",
+  "cohort_vax-sub_age_18_49-dem_lb",
+  "cohort_unvax-sub_age_18_49-dem_lb",
+  "cohort_prevax-sub_age_18_49-dem_lb",
+  "cohort_vax-sub_age_18_49-dem_vasc",
+  "cohort_unvax-sub_age_18_49-dem_vasc",
+  "cohort_prevax-sub_age_18_49-dem_vasc",
+  "cohort_vax-sub_age_18_49-dem_any",
+  "cohort_unvax-sub_age_18_49-dem_any",
+  "cohort_prevax-sub_age_18_49-dem_any",
+  "cohort_vax-sub_age_18_49-dem_alz",
+  "cohort_unvax-sub_age_18_49-dem_alz",
+  "cohort_prevax-sub_age_18_49-dem_alz",
+  "cohort_vax-sub_age_18_49-park",
+  "cohort_unvax-sub_age_18_49-park",
+  "cohort_prevax-sub_age_18_49-park",
   "cohort_unvax-sub_highvascrisk_FALSE-dem_any"
```

Comment:
I am wondering whether we should retry "cohort_unvax-sub_highvascrisk_FALSE-dem_any" now that we have removed the younger people. Even though the unvax cohort is mainly younger people, it is possible that the weirdness in the young people was causing convergence issues.